vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/resources/META-INF/resources/frontend/vaadin-spreadsheet/vaadin-spreadsheet-styles.js

```diff
@@ -1055,6 +1054,339 @@ export const spreadsheetStyles = css`
   .clear-filters-button {
     width: 100%;
   }
+
+  /*
+   * Lumo theme variant styles
+   */
+  :host([theme~="lumo"]) {
+    background-color: var(--lumo-base-color);
+  }
+
+  :host([theme~="lumo"]) .v-spreadsheet .sheet .cell {
+    font-family: var(--lumo-font-family);
+    /* color: var(--lumo-body-text-color); */
+    /* background-color: var(--lumo-base-color); */
```

Comment:
While disabling the `color` and `backround-color` properties fixed the issue with the custom colors being overridden, it didn't fix the underlying problem: the new selectors are more specific than the ones that Spreadsheet generates for the custom formatting.

For instance, take the `Poytakartta.xlsx` sample file in our IT module. It should look like this:
![image](https://github.com/user-attachments/assets/13728af2-55ed-4dbd-91b9-29cb0aeaa664)

But it looks like this:
![image](https://github.com/user-attachments/assets/68333d6d-bac2-48b0-9619-6f9e73e4fbdb)

The issue is the same, but in this case properties like `font-family`, `border-*`, and `font-size` are overridden by the properties defined under this selector because of its specificity. So, either make these selectors less specific (maybe using `:where(:host([theme~="lumo"]))` in the selector) or make the generated selectors more specific.

I also noticed that there's a default style for the cell generated:
```css
.v-spreadsheet.spreadsheet-{HASH-ID} .sheet .cell {
    font-family: Calibri, swiss, Helvetica, arial;
    font-size: 11pt;
    background-color: rgb(255, 255, 255);
    color: rgb(0, 0, 0);
}
```

When I applied the suggestion to make the Lumo styles less specific, then these properties got applied instead of the ones defined by the new Lumo styles for the cells. So that would need to be handled somehow (maybe also generate the default cell style for the Lumo theme?).

Reply:
> using :where(:host([theme~="lumo"])) in the selector)

I chose this to be acceptable compromise.